BootloaderCommonPkg/Library/GraphicsLib/BmpFormat.c

```diff
@@ -233,11 +245,11 @@ ConvertBmpToGopBlt (
         // Other bit format BMP is not supported.
         //
         if (IsAllocated) {
-          // FreePool (*GopBlt);
+          FreePool (*GopBlt);
```

Comment:
can we free the buffer at one place?   Maybe change the flow a little bit to reuse the code at line 268-271 ?

Reply:
Added a Status variable and goto statement, if there is a better way to do it please let me know : )